app/templates/createquiz.html

```diff
@@ -16,8 +16,8 @@ <h1>Create A New Quiz</h1>
             {{ form.op4(size=32, placeholder="Option 4") }}
             {{ form.corrOp(placeholder="Correct Option") }}
             {{ form.img }}
-            <a href="{{ url_for('createquiz') }}" class="route2-btn">Add Another Question</a>
-            <a href="{{ url_for('createquizsuccess') }}" class="route2-btn">Submit Quiz</a>
+            <a href="{{ url_for('createnewquiz') }}" class="route2-btn">Add Another Question</a>
```

Comment:
Button says Add Another Question, but url is for createnewquiz. Very confusing.

Reply:
Will rename to make it clearer